Expertise, Memory, & EIA with other misc tweaks and timing (#59)
data-lessons/instructor-training: _episodes/01-welcome.md

- "What should you expect from this workshop?"
objectives:
- "Identify common ground with some of your fellow workshop participants."
- "Predict what will and will not be covered in this workshop."
- "Know where to find The Carpentries Code of Conduct and how to report an incident."
keypoints:
- "The Carpentries is a community of practice. We strive to provide a welcoming environment for all learners and take our Code of Conduct seriously."
- "This episode sets the stage for the entire workshop. The introductions and exercises help everyone begin to develop a relationship and trust."
- "This workshop will cover evidence-based teaching practices and how they apply specifically to The Carpentries."
- "Learner motivation and prior knowledge vary widely, and can be quickly assessed with a multiple choice question."
---

> ## Pronouns and Names
> 
> Using correct names and pronouns (e.g. "she/her") is important to setting a tone of respect. Learning these is hard to
> do quickly, so we recommend displaying it prominently during the workshop. 
> 
> In an online workshop, give everyone a moment to update their display name to reflect how they would like to be addressed. 
> 

> > At an in-person event, we recommend supplying name tags and low-voc markers, or using plain paper to create table-displayed name placards.

Author: ```suggestion
> At an in-person event, we recommend supplying name tags and markers, or using plain paper to create table-displayed name placards.
```

teaching: 10
exercises: 15
questions:
- "What is The Carpentries and how do we approach teaching?"
- "What should you expect from this workshop?"
objectives:

> - "Identify common ground with some of your fellow workshop participants."

Author: ```suggestion
- "Identify common ground with some of your fellow workshop participants."
- "Understand a general structure and core goals of The Carpentries."
```

- "Predict what will and will not be covered in this workshop."
- "Know where to find The Carpentries Code of Conduct and how to report an incident."

> keypoints:

Reviewer: Learning a bit about The Carpentries still seems to be an objective of this lesson. Maybe it should still be in the learning objectives.

- "The Carpentries is a community of practice. We strive to provide a welcoming environment for all learners and take our Code of Conduct seriously."
- "This episode sets the stage for the entire workshop. The introductions and exercises help everyone begin to develop a relationship and trust."
- "This workshop will cover evidence-based teaching practices and how they apply specifically to The Carpentries."
- "Learner motivation and prior knowledge vary widely, and can be quickly assessed with a multiple choice question."
---

> ## Pronouns and Names
> 
> Using correct names and pronouns (e.g. "she/her") is important to setting a tone of respect. Learning these is hard to
> do quickly, so we recommend displaying it prominently during the workshop. 
> 
> In an online workshop, give everyone a moment to update their display name to reflect how they would like to be addressed. 
> 

> > At an in-person event, we recommend supplying name tags and low-voc markers, or using plain paper to create table-displayed name placards.

Reviewer: I'm not sure what low-voc is, so maybe others won't know either?

Author: I snuck this in there because I had someone at a workshop impress upon me that sharpies were a fairly common asthma trigger and that alternatives existed. I'm not sure it's important enough to merit a definition of "low-voc" though and I'm not sure there's a better alternative. We could just take this out...

Reviewer 2: I also have never heard of low-voc. Maybe we can just say markers

Software Carpentry, Data Carpentry, and Library Carpentry
are official Lesson Programs of The Carpentries.
Together, they form a **global community of volunteer researchers, educators, and others** oriented around
improving basic computing and data skills for
researchers through intensive, short-format workshops.
* Software Carpentry focuses on helping researchers develop foundational
computational skills

> * Data Carpentry focuses on helping

Reviewer: In the section about the main goal, should we say something about developing confidence to continue learning?

Author: This is a good point, but I can't figure out a quick way to address it without zooming out to talk about our 'hidden curriculum.' Which I think we do pretty well at other points, at least. Leaving it alone for now, but feel free to raise this again after merging.

## Instructor Training Workshop Overview

> The goal of this two-day training is to provide you with the skills and information you need

Reviewer: With the experimentation on different schedule formats, do we want to move away from talking about this workshop as a two-day training?

Author: ```suggestion
The goal of this training is to provide you with the skills and information you need
```

Just like learning a new language, a musical instrument,
or a sport, teaching is a skill that requires practice and feedback.
We will have many opportunities to practice and give each other feedback throughout this workshop.

### Creating a Positive Learning Environment

> One part of making this a productive two days for all of us is a

Reviewer: ```suggestion
One part of making this a productive experience for all of us is a
```

is one piece of this. We will also be discussing and practicing teaching techniques to create a positive and
welcoming environment in your classrooms, and will spend some time talking about why this is so important.

### The Carpentries History and Culture

In addition to the teaching practices and philosophy that have been
adopted by The Carpentries community, it is helpful to become familiar 
with our community structure and organisational procedures as you 
prepare to join our Instructor community.
The greatest asset of The Carpentries is people like
you - people who want to help researchers learn new skills 
and share their own experience and enthusiasm.  Meeting your fellow trainees and
Instructor Trainers  at today's event is your first step into The Carpentries community.

## What We Leave Out

We will **not** be going over Data Carpentry, Library Carpentry, or Software Carpentry workshop content in detail (although you will gain
familiarity with some of the content through the exercises),
This workshop is a significant requirement for becoming a certified Carpentries Instructor.

> The additional steps for certification will require that you dig into the workshop content yourself. We will talk about that more tomorrow afternoon.

Reviewer: ```suggestion
The additional steps for certification, called "Checkout" will require that you dig into the workshop content yourself. We will talk about that more tomorrow afternoon.
```

This is an opportunity to introduce the term checkout
